db/db_impl/db_impl.h

```diff
@@ -355,6 +355,12 @@ class DBImpl : public DB {
 
   virtual bool SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) override;
 
+  virtual Status IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                          std::string& ts_low) override;
+
+  virtual Status GetFullHistoryTsLow(ColumnFamilyHandle* column_family,
```

Comment:
Keep `override` and remove `virtual`.

Reply:
Please also add comment about db mutex acquire/release.

Reply:
fixed